source/src/ports/generic_networkhandler.c

```diff
@@ -194,7 +194,7 @@ EipStatus NetworkHandlerInitialize(void) {
 
   /* have QoS DSCP explicit appear on UDP responses to unicast messages */
   if (SetQosOnSocket( g_network_status.udp_unicast_listener,
-                     GetPriorityForSocket(0xFFFF) ) != 0) {
+                     CipQosGetDscpPriority(0xFFFF) ) != 0) {
```

Comment:
do not use magic number

Reply:
as already commented before the code lacks kConnectionObjectPriorityExplicit